testing/unit_tests/t_utils.py

```diff
@@ -20,6 +22,15 @@ def download_data_testing_test_files():
     download_dataset("data_testing")
 
 
+@pytest.fixture(scope='session')
+def download_multi_data():
+    """
+    This Pytest fixture DOWNLOAD all the test data set REQUIRED for the multi-session, multi-contrast related unite
+    testing.
+    """
+    download_dataset("data_multi_testing")
```

Comment:
I think I'm missing something, I don't understand where is this new multi testing data?

Reply:
Good point. I didn't update the DITC_URL for the data. Good catch!

Should be something similar to this in `download_data.py`'s DICT_URL'

![image](https://user-images.githubusercontent.com/6720068/155933138-59f2271a-a315-4e19-9248-c7d850f03475.png)